generators/entity/templates/client/angular/src/main/webapp/app/entities/_entity-management.state.ts

```diff
@@ -47,22 +46,22 @@ export const <%= entityInstance %>State = {
         <%_ if (pagination == 'pagination' || pagination == 'pager'){ _%>
         {
             token: 'pagingParams',
-            deps: [PaginationUtil, '$stateParams'],
-            resolveFn: (paginationUtil, stateParams) => {
+            deps: [PaginationUtil, Transition],
+            resolveFn: (paginationUtil: PaginationUtil, trans: Transition) => {
                 return {
-                    page: paginationUtil.parsePage(stateParams['page']),
-                    sort: stateParams['sort'],
-                    search: stateParams['search'],
-                    predicate: paginationUtil.parsePredicate(stateParams['sort']),
-                    ascending: paginationUtil.parseAscending(stateParams['sort'])
+                    page: paginationUtil.parsePage(trans.params()['page']),
+                    sort: trans.params()['sort'],
+                    search: trans.params()['search'],
+                    predicate: paginationUtil.parsePredicate(trans.params()['sort']),
+                    ascending: paginationUtil.parseAscending(trans.params()['sort'])
                 };
             }
         }<%= (pagination == 'pagination' || pagination == 'pager' && enableTranslation) ? ',' : '' %>
         <%_ } if (enableTranslation){ _%>
         {
             token: 'translate',
             deps: [JhiLanguageService],
-            resolveFn: (languageService) => languageService.setLocations('<%= i18nToLoad %>')
+            resolveFn: (languageService) => languageService.setLocations(['<%= i18nToLoad %>'])
```

Comment:
Probably printing the array wasnt ideal, may be it needs to be concatenated or the array needs to be joined here to form a proper array string